codec/cache/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+description 'Codec Cache: Deephaven Codec Cache'
+
+dependencies {
+    implementation project(":codec-base"),
+                   project(":codec-builtin"),
+                   project(":Base")
```

Comment:
Looks like this dependency is only necessary for a single call to `io.deephaven.base.verify.Require#neqNull(T, java.lang.String)`. Assuming we could remove this dependency and just use `Objects.requireNonNull`?

Reply:
Sure.  I didn't want to go full YOLO here because I wasnt sure how the change was going to be receied.  I did the bare minimum for a prototype tyo throw rocks at.  I'll. incorporate your comments now